src/server/api/controllers/users.controller.js

```diff
@@ -18,6 +18,27 @@ const createUser = async (body) => {
   };
 };
 
+const getUsers = async () => {
+  return knex('users');
+};
+const getUserById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
+  }
+
+  try {
+    const users = await knex('users')
+      .select('users.id as id', 'full_name', 'email', 'address')
+      .where({ id });
+    if (users.length === 0) {
+      throw new Error(`incorrect entry with the id of ${id}`, 404);
```

Comment:
You should define these status responses 400 and 404 in the swagger properties. 
Everything else is working perfectly.

Reply:
Done. Could you check it Divya and lemme know?